src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -96,6 +97,7 @@ public function build(ContainerBuilder $container)
         $this->addCompilerPassIfExists($container, TranslatorPass::class);
         $container->addCompilerPass(new LoggingTranslatorPass());
         $container->addCompilerPass(new AddCacheWarmerPass());
+        $container->addCompilerPass(new CachePoolPrunerPass());
```

Comment:
I follow everything short of "next to CachePoolClearerPass" --- does "next to" mean "before" or "after" CachePoolClearerPass?